addons/web_editor/static/src/xml/editor.xml

```diff
@@ -97,8 +99,9 @@
 
             <div id="justify" class="btn-group dropdown">
                 <button type="button" class="btn dropdown-toggle"
-                    data-bs-toggle="dropdown" tabindex="-1" data-bs-display="static" aria-expanded="false">
+                    data-bs-toggle="dropdown" tabindex="-1" data-bs-original-title="Paragraph" aria-expanded="false">
                     <i id="paragraphDropdownButton" class="fa fa-align-left fa-fw" title="Text align"></i>
+                    <i id="paragraphDropdownButton" class="fa fa-align-left fa-fw"></i>
```

Comment:
You remove this line in the next commit. I'm assuming this is a rebasing error :-)